packages/create-pantheon-decoupled-kit/src/generators/umami.generator.ts

```diff
@@ -0,0 +1,28 @@
+import type { CustomActionConfig } from 'node-plop';
+import type { DecoupledKitGenerator } from '../types';
+
+export const umami: DecoupledKitGenerator = {
+	name: 'umami-demo',
+	description: 'Umami profile data and components',
+	prompts: [
+		{
+			name: 'outDir',
+			message: 'Where should the output go?',
+		},
```

Comment:
this is missing the default output. Check the other generators outDir prompt, it should be the same here.

Reply:
I set a new default for this `outDir` that is based on the other generators `outDir` prompts. It is slightly different since this generator does not take an `appName`